api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
+
+class PlosTaxonomyMixin(JSONAPIBaseView, generics.RetrieveAPIView):
+    permission_classes = (
+    )
+    serializer_class = TaxonomySerializer
+    view_category = 'plos-taxonomies'
+
+    # overrides RetrieveAPIView
+    def get_object(self):
+        with open(self.data_file, 'r') as json_file:
+            data = json.load(json_file)
+        return data
+
+class PlosTaxonomyFlat(PlosTaxonomyMixin):
```

Comment:
Or maybe even just `PlosFlatList` and `PlosTreeviewList` as we know they're taxonomies from the Django app name! Just to make them less unwieldy. Just a suggestion tho!